OpenRA.Mods.Common/Traits/World/ActorMap.cs

```diff
@@ -18,12 +18,19 @@
 
 namespace OpenRA.Mods.Common.Traits
 {
-	public class ActorMapInfo : ITraitInfo
+	public class ActorMapInfo : ITraitInfo, IRulesetLoaded
 	{
 		[Desc("Size of partition bins (cells)")]
 		public readonly int BinSize = 10;
 
+		// The largest hit radius for any actor.
+		public WDist LargestActorRadius { get; private set; }
```

Comment:
This needs a `[FieldLoader.Ignore]`.

Reply:
That's an error because the attribute isn't valid on properties.

Reply:
It might be better to move this to `ActorMap` to avoid confusion.

Reply:
I was thinking to avoid recalculating it more than once, but since this is a `World` trait, you're probably right. It's not going to have many instances constructed over the life of the game like another `Actor` might.

Reply:
Ah wait, but is there a way for the `ActorMap` to get access to the `Ruleset`?

Reply:
Yes, via the `world.Map`.